extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -110,14 +98,29 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
+	function getRecoveredIframe(slotName) {
+		var fallbackId = win._sp_.getElementId(document.querySelectorAll('div[id="' + slotName + '"] div')[1].id);
+		return doc.getElementById(fallbackId).querySelector('div:not(.hidden) > div[id*="_container_"] iframe');
+	}
+
 	function onReady(slotName, callback) {
 		var iframe = doc.getElementById(slotName).querySelector('div:not(.hidden) > div[id*="_container_"] iframe');
 
-		if (!iframe) {
+		if (!iframe && !recoveryHelper.isBlocking()) {
```

Comment:
`!recoveryHelper.isBlocking` doesn't mean iframe doesn't exist, I guess. Create another if with
` if (!recoveryHelper.isBlocking()) {`
and log the message that describe the issue